js/google_ima.js

```diff
@@ -668,8 +668,11 @@ require("../html5-common/js/utils/utils.js");
 
       this.setVolume = function(volume)
       {
-        if (_IMAAdsManager && _linearAdIsPlaying)
+        if (_IMAAdsManager)
         {
+          if (typeof this.savedVolume === "number") {
```

Comment:
Should this be `!==`?

Reply:
Nope, because I'm getting rid of the savedVolume at this point. I technically don't need this type check though. I'll remove it.

Reply:
Oh ok I see